includes/CLI/Plugin_Check_Command.php

```diff
@@ -211,7 +239,7 @@ static function ( $dirs ) use ( $excluded_directories ) {
 		}
 
 		// Get formatter.
-		$formatter = $this->get_formatter( $assoc_args );
+		$formatter = $this->get_formatter( $assoc_args, 'check' );
```

Comment:
See my other comment, I think it would be cleaner to simply pass the actual default fields here.